specification/index.bs

```diff
@@ -154,6 +155,8 @@ Filenames beginning with an underscore (`_`) are reserved for use by user agent.
 
 ## Isolated worlds
 
+<dfn>Worlds</dfn> are isolated JavaScript contexts with access to the same underlying DOM tree but their own set of wrappers around those DOM objects.
```

Comment:
I'm not sure about this definition. In Chrome worlds are main, isolated, and worker. So a main world is never isolated. I don't know what the intent is with regard to the spec, but there's potential for confusion here.